amenity-points.mss

```diff
@@ -4,1894 +4,1243 @@
+      shield-file: url('symbols/cave.svg');
+      shield-fill: black;
+      a/marker-file: url('symbols/cave.svg');
+      a/marker-fill: black;
+    }
+    [feature = 'natural_spring'] {
+      shield-file: url('symbols/spring.svg');
+      shield-fill: @water-text;
+      shield-text-dy: 8;
+      a/marker-file: url('symbols/spring.svg');
+    }
+    [feature = 'man_made_windmill'] {
+      shield-file: url('symbols/windmill.svg');
+      a/marker-file: url('symbols/windmill.svg');
+    }
+    [feature = 'amenity_hunting_stand'] {
+      shield-file: url('symbols/hunting_stand.svg');
+      a/marker-file: url('symbols/hunting_stand.svg');
+    }
+    [feature = 'leisure_miniature_golf'] {
+      shield-file: url('symbols/miniature_golf.p.20.png');
+      shield-fill: darken(@park, 60%);
+      shield-text-dy: 13;
+      a/marker-file: url('symbols/miniature_golf.p.20.png');
+    }
+    [feature = 'leisure_golf_course'] {
+      shield-file: url('symbols/golf.p.20.png');
+      shield-fill: darken(@park, 60%);
+      shield-text-dy: 13;
+      a/marker-file: url('symbols/golf.p.20.png');
+    }
+    [feature = 'aeroway_helipad'] {
+      shield-file: url('symbols/helipad.16.svg');
+      shield-fill: darken(@airtransport, 15%);
+      shield-text-dy: -11;
+      a/marker-file: url('symbols/helipad.16.svg');
+      a/marker-fill: @airtransport;
+    }
+    [feature = 'aeroway_aerodrome'] {
+      shield-file: url('symbols/aerodrome.12.svg');
+      shield-fill: darken(@airtransport, 15%);
+      shield-text-dy: -11;
+      a/marker-file: url('symbols/aerodrome.12.svg');
+      a/marker-fill: @airtransport;
+    }
+    [feature = 'tourism_caravan_site'] {
+      shield-file: url('symbols/caravan_park.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/caravan_park.svg');
+      a/marker-fill: @transportation-icon;
+    }
+    [feature = 'tourism_camp_site'] {
+      shield-file: url('symbols/camping.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/camping.svg');
+      a/marker-fill: @transportation-icon;
+    }
+    [feature = 'leisure_water_park'] {
+      shield-file: url('symbols/water_park.svg');
+      a/marker-file: url('symbols/water_park.svg');
+    }
+    [feature = 'leisure_dog_park'] {
+      shield-file: url('symbols/dog_park.svg');
+      a/marker-file: url('symbols/dog_park.svg');
+    }
+    [feature = 'leisure_playground'] {
+      shield-file: url('symbols/playground.svg');
+      a/marker-file: url('symbols/playground.svg');
 
-  [feature = 'amenity_atm'][zoom >= 17] {
-    marker-file: url('symbols/atm.svg');
-    marker-fill: @amenity-brown;
-    marker-placement: interior;
-    marker-clip: false;
+      [access != ''][access != 'permissive'][access != 'yes'] {
+        shield-opacity: 0.33;
+        shield-text-opacity: 0.33;
+        a/marker-opacity: 0.33;
+      }
+    }
   }
+  [feature  ='amenity_fountain'][zoom >= 17] {
```

Comment:
The problem was the layering of nozzle and pool. It required `marker-allow-overlap` to display both layered above each other. The downside to this approach is that it also allows other markers to overlap with them which happened frequently. (see #1934)